shared/rocroller/lib/source/Expression_generate.cpp

```diff
@@ -187,19 +187,19 @@ namespace rocRoller
              * Evaluates each expression in `exprs`, storing the results in respective indices of
              * `results`.
              *
-             * Each writeable special register may store up to one result. If this is the case,
-             * the scheduler will be locked, and `schedulerLocked` will be set to `true`.
+             * During expression evaluation, special resources may be acquired. If this is the case,
+             * the respective Scheduler::Dependency is to be locked, and `schedulerLockCount` be incremented.
```

Comment:
The voice is a bit odd here, consider something like:
```suggestion
             * During expression evaluation, special resources may be acquired. When this occurs,
             * lock the corresponding Scheduler::Dependency and increment `schedulerLockCount`.
```

Reply:
That's so much better! I had a weird bias of wanting to keep original phrases. Thanks.